oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/mysql-service.json

```diff
@@ -292,32 +292,8 @@
       "layer": "MYSQL",
       "entity": "Service",
       "name": "MySQL-Service",
-      "isRoot": false,
-      "isDefault": true,
-      "expressions": [
-        "avg(meter_mysql_commands_select_rate)",
-        "avg(meter_mysql_commands_insert_rate)",
-        "avg(meter_mysql_commands_update_rate)",
-        "avg(meter_mysql_commands_delete_rate)"
```

Comment:
Why are these removed?

Reply:
@yswdqz Are you using the latest UI version?  It shouldn't delete the hierarchy part when export.